osu.Game.Tests/Visual/Playlists/TestScenePlaylistsResultsScreen.cs

```diff
@@ -42,15 +42,15 @@ public partial class TestScenePlaylistsResultsScreen : ScreenTestScene
         private int totalCount;
         private ScoreInfo userScore;
 
-        [SetUpSteps]
-        public override void SetUpSteps()
+        // We don't use SetUpSteps for this because one of the tests shouldn't receive any scores.
+        public void InitialiseUserScoresAndBeatmap(bool noScores = false)
```

Comment:
Rather than do this `noScores` bool please split the setup of the request handler to a separate method and call it in every test according to the test's need.

Reply:
Not sure I understand. Do you mean only this `noScores` bool? I'm not sure how I would get rid of it below all the way to `createIndexResponse`.
So should I split out just `bindHandler(noScores: noScores);` to something like `SetUpRequestHandler` which also takes `bool noScores = false`?

Reply:
Sure I guess.

Reply:
Makes sense now because now it can go back to using `[SetUpSteps]`.